AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -139,31 +223,150 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
         languageTextField.text = languagesDict[languageSelected]
         regionTextField.text = regionsDict[regionSelected]
-        
+        partialResultsStabilizationTextField.text = partialResultsStabilizationsList[partialResultsStabilizationSelected]
+        piiContentIdentificationTextField.text = piiContentIdentificationDict[piiContentIdentificationSelected]
+        piiContentRedactionTextField.text = piiContentRedactionDict[piiContentRedactionSelected]
+    
         enginePickerView.delegate = self
         languagePickerView.delegate = self
         regionPickerView.delegate = self
         enginePickerView.dataSource = self
         languagePickerView.dataSource = self
         regionPickerView.dataSource = self
+        partialResultsStabilizationPickerView.delegate = self
+        partialResultsStabilizationPickerView.dataSource = self
+        piiContentIdentificationPickerView.delegate = self
+        piiContentIdentificationPickerView.dataSource = self
+        piiContentRedactionPickerView.delegate = self
+        piiContentRedactionPickerView.dataSource = self
         
         enginePickerView.tag = 1
         languagePickerView.tag = 2
         regionPickerView.tag = 3
+        partialResultsStabilizationPickerView.tag = 4
+        piiContentIdentificationPickerView.tag = 5
+        piiContentRedactionPickerView.tag = 6
+        
+        storyBoard = UIStoryboard(name: "Main", bundle: nil)
+        languageDetectionViewController = storyBoard!.instantiateViewController(withIdentifier: "languageDetection") as? AutomaticLanguageDetectionViewController
+        
+        languageDetectionViewController?.cancellationHandler = { numberOfselectedLanguages in
+            self.enableAutomaticLanguageDetectionSwitch.isOn = numberOfselectedLanguages! > 1
+        }
+        
+        languageDetectionViewController?.selectedLanguageOptionsHandler = { selectedLanguages in
+            self.selectedLanguagesDetectionOptions = selectedLanguages!
+        }
+        
+        languageDetectionViewController?.preferredLanguageOptionHandler = { preferredLanguage in
+            self.preferredLanguagesDetectionOptions = preferredLanguage!
+        }
+        
+        self.languageHandler = { languageList in
+            self.languageDetectionViewController?.languages = languageList!
+        }
+        
+        self.languagesDictHandler = { mappings in
+            self.languageDetectionViewController?.languagesDict = mappings!
+        }
     }
     
     @IBAction func startTranscriptionButton(_ sender: UIButton) {
-        let url = self.meetingEndpointUrl.hasSuffix("/") ? self.meetingEndpointUrl : "\(self.meetingEndpointUrl)/"
+        var url = AppConfiguration.url
+        url = url.hasSuffix("/") ? url : "\(url)/"
+        
+        let contentIdentificationEnabled = piiContentIdentificationSelected != ""
+        let contentRedactionEnabled = piiContentRedactionSelected != ""
```

Comment:
isEmpty() - https://developer.apple.com/documentation/swift/string/2946268-isempty

Reply:
will be updated.